ui/src/components/auth/LoginContainer.tsx

```diff
@@ -0,0 +1,141 @@
+import { useDispatch } from 'react-redux';
+import { signin } from '../../reducers/userSlice';
+import axios from 'axios';
+import { Box, Button, Checkbox, Link, Stack, TextField, Typography } from '@mui/material';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { z } from 'zod';
+import { LoginFormFields } from './helpers/form.types';
+import { formSpacing, formWidth } from './helpers/constants';
+import { emailValidator, passwordValidator } from './helpers/zod.validators';
+
+// zod validation
+const loginValidationSchema = z.object({
+  email: emailValidator,
+  password: passwordValidator,
+});
+
+const LoginContainer: React.FC = () => {
+  const { register, handleSubmit, formState } = useForm<LoginFormFields>({
+    resolver: zodResolver(loginValidationSchema),
+  });
+
+  const { errors } = formState;
+
+  const [invalidLogin, setInvalidLogin] = useState<string>('');
+  const [rememberPassword, setRememberPassword] = useState<boolean>(false);
+
+  const dispatch = useDispatch();
+
+  const handleEmailLogin: SubmitHandler<LoginFormFields> = async (data) => {
+    try {
+      const payload = await axios.post('/api/auth/login', {
+        email: data.email,
+        password: data.password ?? '',
+        isRememberPassword: rememberPassword,
+      });
+      dispatch(signin(payload.data));
+      setInvalidLogin('');
+      window.location.reload();
+    } catch (error: any) {
+      if (error?.response?.status === 500 || error?.response?.status === 401) {
+        setInvalidLogin('Invalid email or wrong password');
+      }
+      console.log(error);
+    }
+  };
+
+  return (
+    <>
+      <Box
+        minHeight={'100vh'}
+        sx={{
+          backgroundImage: `url(${process.env.PUBLIC_URL}/images/psalted-bg.jpg)`,
+          backgroundSize: 'cover',
+          backgroundRepeat: 'no-repeat',
+          backgroundAttachment: 'fixed',
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <Box margin={'auto'} bgcolor={'transparent'} width={formWidth}>
+          <Stack direction={'column'} margin={'auto'} spacing={formSpacing}>
+            <Stack spacing={0.5}>
+              <Typography variant="h1" color={'primary'} textAlign={'center'} fontWeight={'bold'}>
+                LOGIN
+              </Typography>
+              <Typography variant={'body1'} textAlign={'center'}>
+                Welcome Back
+              </Typography>
+            </Stack>
+            <form onSubmit={handleSubmit(handleEmailLogin)}>
+              <Stack spacing={formSpacing}>
+                <Stack spacing={1}>
+                  <TextField
+                    label="Email"
+                    fullWidth
+                    autoComplete={'email'}
+                    autoFocus
+                    {...register('email', {
+                      required: 'Required',
+                    })}
+                  />
+                  {errors.email && (
```

Comment:
i find out you can do this (i.e. add `error` and `helperText` keys( to handle error in `TextField`
<img width="430" alt="image" src="https://github.com/hmcc-global/psalted-2/assets/26876443/37a3bab8-42b3-4e8a-94b0-62d764e0fe7f">
so you don't have to add another Typography component